### ISSUE TITLE
Allow installing with pip

### ISSUE THREAD

**Issue by mrfuxi** [getpelican/pelican-plugins#213]
It would be great if plugins could be installed with pip

**Comment by lioman** [comment]
It could be encouraged by only letting pip installable plugins into the catalog

**Comment by moorepants** [comment]
Agreed, it is so much easier to manage if plugins are all installable by package managers. It would enable pinning plugin versions, dependency resolution, remove the need to clone a giant repository of many plugins, and just make it much easier to manage plugins (ant themes!).
